### PR TITLE
Invalid stats report request causes 500 return code

### LINKED ISSUE #3229: Invalid stats report request causes 500 return code
Request: http://www.nuget.org/stats/packages/635000865762634438AggregateStatsInHomePage/1.0.0

### PR DESCRIPTION
**Pull request by maartenba**
Fixes #3229

@scottbommarito @xavierdecoster @skofman1 @shishirx34

### CODE REVIEW

**Comment by xavierdecoster** [comment]
:shipit:

**Comment by skofman1** [comment]
:shipit:

File: src/NuGetGallery/Controllers/StatisticsController.cs [hunk 1]
```diff
@@ -148,16 +161,17 @@ public StatisticsController(IStatisticsService statisticsService, IAggregateStat
...
-                Constants.StatisticsDimensions.ClientVersion,
-                Constants.StatisticsDimensions.Operation
-            };
+                report = await _statisticsService.GetPackageDownloadsByVersion(id);
 
-            StatisticsPackagesReport report = await _statisticsService.GetPackageDownloadsByVersion(id);
-            ProcessReport(report, groupby, dimensions, id);
+                ProcessReport(report, groupby, PackageDownloadsByVersionDimensions, id);
+            }
+            catch (StatisticsReportNotFoundException)
```
Comment (scottbommarito) [inline src/NuGetGallery/Controllers/StatisticsController.cs]: Is `StatisticsReportNotFoundException` thrown when the report is invalid, or just when the report is not found?

**Comment by shishirx34** [comment]
Looks good! :shipit:

**Comment by scottbommarito** [comment]
In what situation are we getting an invalid stats report?

**Comment by maartenba** [comment]
See #3229 :)

**Comment by scottbommarito** [comment]
lgtm

**Pull request merged**